submissions/kolawoleOgunfowokan/HTML/index.html

```diff
@@ -307,5 +371,6 @@ <h3 class="footer-text">CONNECT WITH US</h3>
             <h4 class="footer-copyright">Copyright &copy 2019 myoja.com. All rights reserved</h4>
         </section>
     </footer>
+    <script src="/js/script.js"></script>
```

Comment:
The JS file was not loaded when I tried to view this page. `js/script.js` should be fine

Reply:
Fixed. I used "js/script.js"